### PR TITLE
Customizable PV info - Added missing mongo PVC

### PR DESCRIPTION
**Pull request by zagganas**
## Description

This PR provides a mongo PVC that was previously missing from the deployment. Also made PV/PVC information more customizable.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

## Checklist:

- [ ] My code follows the [style guidelines](https://github.com/elixir-cloud-aai/elixir-cloud-aai/blob/dev/resources/contributing_guidelines.md#language-specific-guidelines) of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] New and existing unit tests pass locally with my changes
- [ ] I have not reduced the existing code coverage
- [ ] I have updated any sections of the app's documentation that are affected by the proposed changes, if applicable

### CODE REVIEW

File: deployment/templates/mongodb/mongodb-pvc.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+   type: local
+spec:
+ storageClassName: {{ .Values.volumes.mongodb.storageClass }}
+ capacity:
+   storage: {{ .Values.volumes.mongodb.size }}
+ accessModes:
+   - ReadWriteOnce
+ hostPath:
+   path: {{ .Values.pathToLocalDir }}
+{{ end }}
```
Comment (lvarin) [inline deployment/templates/mongodb/mongodb-pvc.yaml:16]: PVs will probably not work in OpenShift. They are created automatically when the PVC is created.

Comment (lvarin) [reply deployment/templates/mongodb/mongodb-pvc.yaml:16]: But as Kaniko does not currently work in Openshift, it is not a very important issue.

**Review by lvarin — COMMENTED** [review]
LGTM!

@akash2237778 what do you think?

File: deployment/templates/mongodb/mongodb-pvc.yaml (hunk 1 above)
Comment (zagganas) [reply deployment/templates/mongodb/mongodb-pvc.yaml:16]: At the moment you can set the deploy PV value to false and it won't get deployed. Instead you can use an existing RWO StorageClass for the PVC.

Comment (lvarin) [reply deployment/templates/mongodb/mongodb-pvc.yaml:16]: 👍🏽

**Comment by akash2237778** [comment]
Also looks fine to me. Thanks @zagganas. Also can you please squash it to one commit?

**Review by lvarin — APPROVED** [review]
lgtm

**Comment by lvarin** [comment]
I will squash the commits when merging.

**Pull request merged**